### ISSUE TITLE
1.4.0: `iframe.contentWindow` undefined is not object errors

### ISSUE THREAD

**Issue by machty** [testem/testem#798]
Upgrading to testem 1.4.0 seems to break things on ember-cli: https://travis-ci.org/machty/ember-concurrency/jobs/111267517

```
not ok 1 PhantomJS 2.1 - Global error: TypeError: undefined is not an object (evaluating 'iframe.contentWindow') at http://localhost:7357/testem.js, line 819
    ---
        Log: |
            { type: 'error',
              text: 'TypeError: undefined is not an object (evaluating \'iframe.contentWindow\') at http://localhost:7357/testem.js, line 819\n' }
```

Downgrading to 1.3.0 fixes this issue.

/cc @rwjblue and @abuiles

**Comment by johanneswuerbach** [comment]
Looks like a race condition when the iframe is not embedded fast enough.

**Comment by rwjblue** [comment]
@johanneswuerbach - Agreed.  Unless someone else tackles this first, I'll try to dig into it tonight...

**Comment by brancusi** [comment]
Getting this as well on any change after initial startup.

**Comment by bantic** [comment]
I will tackle this tonight if @rwjblue doesn't beat me to it. Should be a quick fix.

**Comment by johanneswuerbach** [comment]
Is any of you using `postMessage` in their own app? This is a quick fix, which seems to solve the issue for ember-concurrency: https://github.com/testem/testem/pull/799

**Comment by bantic** [comment]
Looks like ember-concurrency uses [`postMessage` via a dependency on Rx.js](https://github.com/machty/ember-concurrency/blob/master/vendor/dummy-deps/rx.js#L1297-L1307). I've confirmed that's where the too-early messages that trigger the `iframe.contentWindow` problem come from.

**Comment by machty** [comment]
Interesting, I think that's where @abuiles also saw these issues coming from.

**Comment by johanneswuerbach** [comment]
Published as `v1.4.1`
